### ISSUE TITLE
[CSS]: Button Alignment Main Page

### ISSUE THREAD

**Issue by sparsh-99** [opensourcedesign/opensourcedesign.github.io#285]
# When we see the main page in responsive view, the alignment of 'Become a sponsor' button is somewhat 'left' to the page

#### Steps to Reproduce:
* Switch to Mobile view
* Scroll to supporter's section
* See the button

#### Screenshot:

![Screenshot (487)](https://user-images.githubusercontent.com/56729873/83335204-a72b1980-a2c8-11ea-837a-f0397a49a480.png)

#### Additional Context:
If you see this as a genuine issue, I would love to work on it.

**Comment by Yashank18** [comment]
Hey @sparsh-99 

I understand your issue. Can you please explain a lit bit more because what I think is the button here is actually an image and its width and height are controlled from the same.
Do let me know if I am not able to understand your point

**Comment by sparsh-99** [comment]
### Look the margin below, you will get the idea-

![Screenshot (511)](https://user-images.githubusercontent.com/56729873/84698150-f0d45f00-af6c-11ea-9c98-29410fb7efed.png)

##### Image is left-indent!!! @Yashank18

**Comment by sparsh-99** [comment]
Will you please help me how can I fix this indentation myself? How can I find this file in this bid code?

**Comment by Yashank18** [comment]
Sure @sparsh-99 
You will find this in Index.html only.
Under div with class="col sm-12 supporters"

**Comment by A-GHOSH-dev** [comment]
Can I contribute here?

**Comment by jdittrich** [comment]
@A-GHOSH-dev yes, I think the next action would be reviewing @sparsh-99 pull request #290 and telling if the code is fine and has the intended results (resolves the problem; for this you probably also need to check out a pre- #290  commit to see the issue)

**Comment by anubhav1206** [comment]
Can i work on this issue?

**Comment by Aditya-Naresh** [comment]
Is this issue still open?

**Comment by anniepauline** [comment]
Willin got work on it
